package.json

```
@@ -4,9 +4,10 @@
   "version": "1.3.3",
   "license": "apache-2.0",
   "scripts": {
+    "build:sdk": "microbundle --compress -f es -o build -i src/sdk.ts",
```

I think `--compress` is the default and `-i` is optional.

```suggestion
    "build:sdk": "microbundle -f es -o build src/sdk.ts",
```

---

This seems to be adding a load of definition files into the build.

---

True. Do you think that’s a problem? There’s technically no harm in emitting them and I have no easy way to teach `microbundle` to _not_ do that.